compyle/opencl.py

```diff
@@ -2,14 +2,12 @@
 """
 from __future__ import print_function
 import pyopencl as cl
-from collections import defaultdict
-from operator import itemgetter
 
 from .config import get_config
+from .profile import profile_kernel, record_profile, named_profile
```

Comment:
Is `named_profile` used anywhere or is record profile used?

Reply:
In pysph `named_profile` is imported from `compyle.opencl` so I added it here. I can change all imports in pysph so that they import from `compyle.profile` in a later PR

Reply:
OK.